### PR TITLE
Populate wordBank with targetLanguage

### PR DESCRIPTION
**Pull request by klappy**
#### This pull request addresses:

This uses the targetLanguage to populate the wordBank in the word alignment tool.

#### How to test this pull request:

Open a compatible project in the wordAlignment tool (after chapter index PR is merged, this can be any titus project). 

Ensure that the word bank is showing the target language words instead of ULB. 

The users will be opening the ULB as their targetLanguage to align.

---
This change is [Reviewable](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2816)

### CODE REVIEW

**Comment by mannycolon** [comment]
:lgtm:

---

Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/2816#-:-KvSkwtrfO0lU0Gnink1:bnfp4nl)*

**Comment by klappy** [comment]
Will need to be reviewed again. Made some updates for some broken references to WordAlignmentHelpers due to git casing. Also renamed a function to the new TopWords instead of TargetWords.

---







*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/2816#-:-KvTBM1mN2vrA2ZjR9a9:b-t28yfv)*

**Comment by RoyalSix** [comment]
Reviewed 6 of 6 files at r3.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/2816)*

File: src/js/actions/WordAlignmentActions.js [hunk 1]
```diff
@@ -2,15 +2,15 @@
 import consts from '../actions/ActionTypes';
 import isEqual from 'lodash/isEqual';
 // helpers
-import * as wordAlignmentHelpers from '../helpers/wordAlignmentHelpers';
+import * as WordAlignmentHelpers from '../helpers/WordAlignmentHelpers';
```
Comment (RoyalSix) [inline src/js/actions/WordAlignmentActions.js:5]: This file has been deleted so the import fails @klappy

**Comment by klappy** [comment]
*[src/js/actions/WordAlignmentActions.js, line 5 at r3](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/2816#-KvT4JFf_UvWbcURhUbK-r3-5:-KvTaV8Fe5Y3hK45Jp1C:b-rbp9d1) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/42944c19ff687af5f3c6e14f24f21451a9f5d0bf/src/js/actions/WordAlignmentActions.js#L5)):*
<details><summary><i>Previously, RoyalSix (Sterling (Jay) Scott) wrote…</i></summary><blockquote>

This file has been deleted so the import fails @klappy 
</blockquote></details>

So, the only way to get this to fully work is on a case-sensitive fs. I had the same issue until I did that.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/2816)*

**Comment by klappy** [comment]
*[src/js/actions/WordAlignmentActions.js, line 5 at r3](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/2816#-KvT4JFf_UvWbcURhUbK-r3-5:-KvTajRrSwuTC-IOiW0j:bgkr7vt) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/42944c19ff687af5f3c6e14f24f21451a9f5d0bf/src/js/actions/WordAlignmentActions.js#L5)):*
<details><summary><i>Previously, klappy (Christopher Klapp) wrote…</i></summary><blockquote>

So, the only way to get this to fully work is on a case-sensitive fs. I had the same issue until I did that.
</blockquote></details>

There should be a file of the same name with a capital W.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/2816)*

**Comment by klappy** [comment]
*[src/js/actions/WordAlignmentActions.js, line 5 at r3](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/2816#-KvT4JFf_UvWbcURhUbK-r3-5:-KvTatBYou5JNqut6H2V:b-eggrj6) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/42944c19ff687af5f3c6e14f24f21451a9f5d0bf/src/js/actions/WordAlignmentActions.js#L5)):*
<details><summary><i>Previously, klappy (Christopher Klapp) wrote…</i></summary><blockquote>

There should be a file of the same name with a capital W.
</blockquote></details>

Let me see what I can do... Somehow github is showing the removal and not the renaming.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/2816)*

**Comment by klappy** [comment]
*[src/js/actions/WordAlignmentActions.js, line 5 at r3](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/2816#-KvT4JFf_UvWbcURhUbK-r3-5:-KvTbJlK8U_w2Y_QrkdR:bg0gt19) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/42944c19ff687af5f3c6e14f24f21451a9f5d0bf/src/js/actions/WordAlignmentActions.js#L5)):*
<details><summary><i>Previously, klappy (Christopher Klapp) wrote…</i></summary><blockquote>

Let me see what I can do... Somehow github is showing the removal and not the renaming.
</blockquote></details>

Here's the commit that renames the file: b064c33

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/2816)*

**Comment by klappy** [comment]
*[src/js/actions/WordAlignmentActions.js, line 5 at r3](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/2816#-KvT4JFf_UvWbcURhUbK-r3-5:-KvTcuQhWEBVLKIVnM3D:bx62s73) ([raw file](https://github.com/unfoldingword-dev/translationcore/blob/42944c19ff687af5f3c6e14f24f21451a9f5d0bf/src/js/actions/WordAlignmentActions.js#L5)):*
<details><summary><i>Previously, klappy (Christopher Klapp) wrote…</i></summary><blockquote>

Here's the commit that renames the file: b064c33
</blockquote></details>

@royalsix Can you confirm that the file is missing on your system or isn't just a bad representation in this review tool?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/unfoldingword-dev/translationcore/2816)*

**Pull request merged**